contracts/strategies/allocation/quadratic-voting-simple/README.md

```diff
@@ -0,0 +1,39 @@
+Spec: Simple Quadratic Voting (QV)
+---------------------------------
+
+## Overview 
+Quadratic Voting is a popular method for democratic decision-making. This strategy implements a simple version, where an allowlist of wallet addresses are given an equal number of voice credits with which to vote. The pool's funds are distributed proportionally to the number of votes each project receives. 
+
+## Spec
+### Recipient logic
+In this strategy, prospective recipients need to apply and be approved by at least two admins. The pool admins can set a time window in which applications must be received.
```

Comment:
What's the reason for having 2 manager ? 
A pool can be created with 1 manager. Could we explore sticking to single approval ?